tests/appsec/iast/aspects/test_add_aspect.py

```diff
@@ -16,7 +15,7 @@ def setup():
     "obj1, obj2",
     [
         (3.5, 3.3),
-        (complex(2, 1), complex(3, 4)),
+        # (complex(2, 1), complex(3, 4)),
```

Comment:
why is this commented out?

Reply:
This PR is part of a larger one, https://github.com/DataDog/dd-trace-py/pull/6283, where this test is working. However, when we split the PR, only `complex` is now broken in this PR. Since IAST has not been published yet, we are being pragmatic and restoring the test in the larger PR